Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -30,7 +29,22 @@ void RuntimeDecorator::decorateRuntime(
   rt.global().setProperty(
       rt, "_LABEL", jsi::String::createFromAscii(rt, label));
 
+  auto dump = [](jsi::Runtime &rt,
+                 const jsi::Value &thisValue,
+                 const jsi::Value *args,
+                 size_t count) -> jsi::Value {
+    auto stats = rt.instrumentation().getRecordedGCStats();
+    Logger::log(stats.c_str());
+    return jsi::Value::undefined();
+  };
+
   jsi::Object dummyGlobal(rt);
+  dummyGlobal.setProperty(rt, "gc", rt.global().getProperty(rt, "gc"));
+  dummyGlobal.setProperty(
```

Comment:
Is `dump` just temporary function for development only or you want to remove it before the merge of PR?